website/release_announcement_drafts/v1.3.4.md

```diff
@@ -6,7 +11,26 @@ started using this build:
 
 - https://jbrowse.org/jb2/docs/tutorials/embed_linear_genome_view/01_introduction
 
-A preliminary beta version of this build was used in a tutorial at BOSC2020,
-and that tutorial was also on our website. If you've used that version, the
+A preliminary beta version of this build was used in a tutorial at BOSC2020, and
+that tutorial was also on our website. If you've used that version, the
 interface has changed somewhat, so come check out the new tutorial for the
 official released version!
+
+## Bookmark widget
+
+A new widget has been added that is accessible from the Linear Genome View view
+menu or when you click and drag over a region in the header (a.k.a rubber band
+selection). This view keeps a list of bookmarked regions, which you can add
+custom labels to, and also which you can use to navigate back to that region.
+The bookmarks can also be exported.
+
+![Bookmark widget in use](https://user-images.githubusercontent.com/19295181/130518189-d8fa8904-d52f-45b0-8403-34f08c23740e.gif)
+
+## Note to plugin developers
+
+If your plugin adds menu items or context menu items to a track, there may be
```

Comment:
little add-on but maybe mention renderProps would also be affected. viewMenu, which wasn't as affected by composition, has a little backcompat